galaxy/values.yaml

```
@@ -267,6 +267,53 @@ extraEnv: []
   # - name: EXAMPLE_ENV
   #   value: MY_VALUE
 
+#- User defined CronJobs
+cronJobs:
+  chown:
+    schedule: "0 * * * *"
+    securityContext:
+      runAsUser: 0
+    command:
+      - /usr/bin/chown
+    args:
+      - "-R"
+      - "galaxy"
+      - "/galaxy/server/database"
+  maintenance:
+    schedule: "5 2 * * *"
+    extraSettings:
+      days: 7
+    securityContext:
+      runAsUser: 0
+    command:
+      - "/bin/bash"
+      - "-c"
+    args:
+      - "/galaxy/server/scripts/maintenance.sh"
+      - "--no-dry-run"
+      - "--days"
+      - "{{ .Values.cronJobs.maintenance.extraSettings.days }}"
+  #- Remove files from the tmp directory that are older than the allowable wall time for a job
+  tmp:
+    schedule: "15 2 * * *"
+    extraSettings:
+      seconds: 604800
```

The original code had a minus in front?
```suggestion
      seconds: -{{ (index .Values "configs" "job_conf.yml" "runners" "k8s" "k8s_walltime_limit" | default 604800) }}
```

---

The leading minus sign (less than) is not required here as that is what `-newermt` does anyway.  At least that is my understanding of the `find` documentation.  ChatGPT agrees (FWIW).